test/templates/alert/component.njk

```diff
@@ -0,0 +1,16 @@
+{#
+    see ./data.json for object structure
+#}
+
+{% from "../button/component.njk" import button %}
+{% macro alert(properties={}, modifier={}) -%}
+<div class="tn-alert{{ modifier.block | modifier('alert') }}" role="alert">
+  <div class="tn-alert__action">
+    {{ button({ icon: 'remove' }, modifier={ block: ['icon', 'text', 'small'] }, aria={ label: 'Remove' }) }}
```

Comment:
To simplify this and avoid the button styling issue, you could add a custom button element with an icon ...

```
<button class="tn-alert__close" aria-controls="{{properties.id}}">{{ icon('remove') }}</button>
```

You'll have to import icon instead of button `{% from "../icon/component.njk" import icon %}`. 

And consider putting an `id="{{properties.id}}"` on the alert with the close button using `aria-controls="{{properties.id}}"`.

> The id could also be randomly generated. It's important that they match but have no significance to a developer so passing it explicitly simply adds overhead.